base/tuple.jl

```diff
@@ -267,6 +267,11 @@ _totuple(::Type{Tuple}, itr, s...) = (collect(Iterators.rest(itr,s...))...,)
 
 end
 
+## construction with one element ##
+
+Tuple(x::Ref) = tuple(getindex(x))
+Tuple(x::Array{T,0}) where {T} = tuple(getindex(x))
```

Comment:
These should be inside the above `if` block, at the top with the other constructors.

Reply:
OK, I’ve moved them upwards (but left the definitions unchanged).